### PR TITLE
Code challenge solution #1.5

### PR DESCRIPTION
**Pull request by ehats**
This time into the right branch

### CODE REVIEW

**Comment by callebjorkell** [comment]
In the future, good commit messages: http://chris.beams.io/posts/git-commit/ 
Yours aren't bad, but they could be even better :wink:

**Comment by callebjorkell** [comment]
Please also make the docker image build when you run `mvn package` (or later phase) and use the artifact (war file) in the docker image instead of building it inside of the same.

**Comment by callebjorkell** [comment]
When doing pull requests, please add a description of what the overall functionality is, what has changed how it can be used and anything else one needs to know about the changes in the diff. Also mention the person you want to review so that they get notifications about it. Do this by typing for example `@callebjorkell` in the PR description, if you want to do a FYI for someone, you can also do that by `CC: @jesperterkelsen` for example. 

When a PR is "approved", people will add a comment with a single random emoji to give their stamp of approval.

**Comment by ehats** [comment]
@callebjorkell I changed the way the docker image is built. Now it's part of the build process, which will build the image using the dockerfile, add the compiled war and then push it to the local repo.

File: Riddler/docker/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,6 @@
+FROM ubuntu:trusty
+RUN sudo apt-get update
+RUN sudo apt-get install -y maven default-jdk
```
Comment (callebjorkell) [inline Riddler/docker/Dockerfile]: So when I start this docker image, I don't actually have anything running on 8080 right? No jetty, no web service, just an empty docker image with maven installed. Unless the `docker-maven-plugin` does some magic that I'm unaware of that is.

In other words:
* When you do `mvn package` in the project, you should produce the docker image (you're doing this now)
* When you do `docker start` there should be a container started that runs jetty on port 8080, and that has your web app running.

File: Riddler/pom.xml [hunk 2]
```diff
@@ -0,0 +1,101 @@
...
+            <artifactId>spring-context</artifactId>
+            <version>4.1.6.RELEASE</version>
+        </dependency>
+        <dependency>
+            <groupId>org.glassfish.jersey.ext</groupId>
+            <artifactId>jersey-spring3</artifactId>
+            <version>2.8</version>
+        </dependency>
+        <dependency>
+            <groupId>org.glassfish.hk2</groupId>
```
Comment (callebjorkell) [inline Riddler/pom.xml]: Hmm... what are you using this for?

**Comment by ehats** [comment]
@callebjorkell Removed unneeded dependencies and switched to a standalone Jetty installation

**Comment by callebjorkell** [comment]
:racehorse:

**Pull request merged**